src/Textarea/Textarea.tsx

```diff
@@ -12,8 +12,6 @@ type BaseProps = {
   id: string
   /** className attribute to apply classses from props */
   className?: string
-  /** ref attribute for input */
-  ref?: RefObject<HTMLTextAreaElement>
```

Comment:
Adding it to the props shouldn't do anything, I've wrapper the component in `forwardRef` instead